### PR TITLE
Remove Windows camera cpp inline function 

### PR DESCRIPTION
**Pull request by qiulin**
Description
Reference issue
Parttial fixes https://github.com/pion/mediadevices/issues/347, remove inline function in cpp, which will cause compiling problem when debug

### CODE REVIEW

**Comment by qiulin** [comment]
@edaniels Hi, Can you review the code? Thanks.

File: pkg/driver/camera/camera_windows.cpp [hunk 1]
```diff
@@ -347,6 +359,7 @@ int openCamera(camera* cam, const char** errstr)
   if (src == nullptr || dst == nullptr ||
       FAILED(hr = graphBuilder->Connect(src, dst)))
   {
+    printErr(hr);
```
Comment (edaniels) [inline pkg/driver/camera/camera_windows.cpp:362]: does the error need to be printed here?

File: pkg/driver/camera/camera_windows.go [hunk 2]
```diff
@@ -37,7 +37,7 @@ func init() {
 
 // Initialize finds and registers camera devices. This is part of an experimental API.
 func Initialize() {
-	C.CoInitializeEx(nil, C.COINIT_MULTITHREADED)
+	C.CoInitializeEx(nil, C.COINIT_APARTMENTTHREADED)
```
Comment (edaniels) [inline pkg/driver/camera/camera_windows.go:40]: What is this change?

File: pkg/driver/camera/camera_windows.cpp (hunk 1 above)
Comment (qiulin) [reply pkg/driver/camera/camera_windows.cpp:362]: It's just printing error for debug, I will resolve it

File: pkg/driver/camera/camera_windows.go (hunk 2 above)
Comment (qiulin) [reply pkg/driver/camera/camera_windows.go:40]: It's recommended to work with some UI framework, like Wails (https://wails.io/).

Comment (qiulin) [reply pkg/driver/camera/camera_windows.go]: I reverted it, and will open another PR.

Comment (qiulin) [reply pkg/driver/camera/camera_windows.go]: My English is bad, please forgive any mistakes.

**Review by edaniels — APPROVED** [review]
LGTM

**Pull request merged**